### PR TITLE
Add py3 support

### PR DESCRIPTION
**Pull request by treebee**

### CODE REVIEW

File: httsleep/main.py [hunk 1]
```diff
@@ -32,7 +37,7 @@ def __init__(self, url_or_request, until=None, error=None,
         if ignore_exceptions:
             self.ignore_exceptions = tuple(ignore_exceptions)
         else:
-            self.ignore_exceptions = (None,)
+            self.ignore_exceptions = (_DummyException,)
```
Comment (kopf) [inline httsleep/main.py]: What about `self.ignore_exceptions = tuple()`? Don't know why I didn't do it like this originally...

**Comment by MathMagique** [comment]
Two general remarks:
- I would have expected the `six` python compatibility package to be used. Why do you use custom implementations?
- I would prefer a travis build for both Python 2 and 3. There is not too much point in widening the audience without having continuous integration ;-).

**Comment by kopf** [comment]
> I would prefer a travis build for both Python 2 and 3. There is not too much point in widening the audience without having continuous integration ;-).

Travis CI integration is on the TODO list (see the readme). That'll be added in its own branch.

**Comment by treebee** [comment]
> I would have expected the six python compatibility package to be used. Why do you use custom implementations?

I just don't see the need to pull in a dependency like six for such small compatibility needs especially since six sometimes does things one does not want to.
Also, it's not that uncommon to add an own _compat.py module (e.g. sqlalchemy, flask+other pallets projects).

**Pull request merged**